_layouts/home.html

```diff
@@ -12,20 +12,17 @@ <h1>About</h1>
   {%- if site.events.size > 0 -%}
     <h2 class="post-list-heading">{{ page.list_title | default: "Next Event" }}</h2>
     <ul class="post-list">
-      {% assign sorted = (site.events | where_exp: "item", "item.date >= site.time" | sorted:"date") %}
-      {%- assign event = sorted.first -%}
-      <li>
-        {%- assign date_format = site.minima.date_format | default: "%b %-d, %Y" -%}
-        <span class="post-meta">{{ event.date | date: date_format }}</span>
-        <h3>
-          <a class="post-link" href="{{ event.url | relative_url }}">
-            {{ event.title | escape }}
-          </a>
-        </h3>
-        {%- if site.show_excerpts -%}
-          {{ event.excerpt }}
-        {%- endif -%}
-      </li>
+      <div id="upcoming-event">
+        {%- for event in site.events -%}
+          <div class="event-link" date="{{ event.date }}">
```

Comment:
What uses this? If you need to add ad hoc data, I think `data-` attributes are preferred.

Reply:
This is specifically for the Javavscript to access dynamically

Reply:
Right, but if you're adding ad-hoc data to an element, it should be in a `data-` attribute: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes